nixos/modules/security/pam.nix

```diff
@@ -36,14 +36,61 @@ let
         '';
       };
 
-      u2fAuth = mkOption {
-        default = config.security.pam.enableU2F;
-        type = types.bool;
-        description = ''
-          If set, users listed in
-          <filename>~/.yubico/u2f_keys</filename> are able to log in
-          with the associated U2F key.
-        '';
+      U2F = {
```

Comment:
https://github.com/Yubico/pam-u2f Yubico themselves seem to have no trouble keeping it lowercase